dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/entity/ProcessInstance.java

```diff
@@ -77,6 +74,7 @@ public class ProcessInstance {
     /**
      * end time
      */
+    @TableField(updateStrategy = FieldStrategy.IGNORED)
```

Comment:
Please don't use this way, because it's a hide way for upper callers, will cause other problems if users don't know.

Reply:
emm, but also we cannot config the global field-strategy, may we can add a update sql in mapper.xml?
Or just update the date to 1970, but need change many files.

Reply:
> Please don't use this way, because it's a hide way for upper callers, will cause other problems if users don't know.

emm, but also we cannot config the global field-strategy, may we can add a update sql in mapper.xml?
Or just update the date to 1970, but need change many files.